lib/types/inet-address.js

```diff
@@ -1,247 +1,282 @@
 "use strict";
 
 const utils = require("../utils");
+const rust = require("../../index");
 
 /** @module types */
 /**
- * Creates a new instance of InetAddress
- * @class
- * @classdesc Represents an v4 or v6 Internet Protocol (IP) address.
- * @param {Buffer} buffer
- * @constructor
+ * Represents an IP address.
  */
-function InetAddress(buffer) {
-    if (
-        !(buffer instanceof Buffer) ||
-        (buffer.length !== 4 && buffer.length !== 16)
-    ) {
-        throw new TypeError("The ip address must contain 4 or 16 bytes");
-    }
+class InetAddress {
+    #internal;
```

Comment:
```suggestion
    /**
    * @type {rust.InetAddressWrapper}
    * @private
    */
    #internal;
```

Reply:
Why do you want to add`@private` tag, when it is already marked private by `#`

Reply:
Then at least add ``@type`` tag